src/runtime/test/manifest-integration-test.ts

```diff
@@ -35,14 +34,20 @@ describe('manifest integration', () => {
     await arc.instantiate(recipe);
     await arc.idle;
     const type = recipe.handles[0].type;
-    const [store] = arc.findStoresByType(type);
+    let store;
+    [store] = arc.findStoresByType(type);
     assert(store);
+
+    // TODO(lindner): mismatched type..
+    const proxy = store as StorageProxy;
```

Comment:
aha, the 'any' type used above is sufficient to avoid type clashes.  I will switch to using the proxy object to illustrate the casting behavior that is going on.